package.json

```diff
@@ -1,71 +1,68 @@
 {
   "name": "@grammyjs/nestjs",
-  "description": "grammY module for NestJS",
-  "version": "1.0.1",
-  "author": {
-    "name": "Mike Crowe",
-    "email": "drmikecrowe@gmail.com"
+  "version": "2.0.0",
+  "license": "MIT",
+  "scripts": {
+    "lint:fix": "nx affected:lint --fix",
+    "lint": "nx run-many --all --target=lint",
+    "prepare": "husky install",
+    "test": "nx test",
+    "publish:version": "nx run grammy-nestjs:version",
+    "publish:npm": "nx run grammy-nestjs:deploy --no-build"
   },
   "bugs": "https://github.com/grammyjs/nestjs/issues",
-  "contributors": [
-    {
-      "name": "Arthur Asimov",
-      "email": "arthur.asimov.z0@gmail.com"
-    }
-  ],
   "dependencies": {
-    "@nestjs/common": "^9.0.11",
-    "@nestjs/core": "^9.0.11",
-    "@nestjs/platform-express": "^9.0.11",
-    "debug": "^4.3.4",
+    "@nestjs/common": "^10.0.2",
+    "@nestjs/core": "^10.0.2",
+    "@nestjs/platform-express": "^10.0.2",
+    "axios": "^1.0.0",
     "firebase": "^9.9.3",
     "firebase-admin": "latest",
     "firebase-functions": "latest",
-    "grammy": "^1.11.0",
-    "lodash": "^4.17.21",
+    "grammy": "^1.19.0",
```

Comment:
Should grammY be made a peer dep so that users can install their own version of it?

Reply:
yeah, but this is a worksapce package json, it's already done in grammy-nestjs/package.json

Reply:
I see! Then nevermind